### PR TITLE
Prep for RC scale

### PR DESCRIPTION
**Pull request by TKDMaster12**
@Mandalorian007 
Small style changes
Set up for RC scale

### CODE REVIEW

File: src/main/resources/static/built/css/fixes.css [hunk 1]
```diff
@@ -1,3 +1,3 @@
 .rc-slider-tooltip {
-    z-index: 1000;
+    z-index: 2000;
 }
```
Comment (Mandalorian007) [inline src/main/resources/static/built/css/fixes.css]: can you tack an extra line on the bottom of this file?

Comment (TKDMaster12) [reply src/main/resources/static/built/css/fixes.css]: I can add an extra line on the bottom. What do you want to add?

Comment (Mandalorian007) [reply src/main/resources/static/built/css/fixes.css]: was just a style preference to have 1 new line return at the bottom of each file.  Sorry nitpicky stuff

Comment (TKDMaster12) [reply src/main/resources/static/built/css/fixes.css]: Sorry, I am not understanding what you want. 

You want to add a line to each file to return the css?

Comment (Mandalorian007) [reply src/main/resources/static/built/css/fixes.css]: no just a new line at the end of the file.  It stops github from putting an annoying red symbol

**Comment by TKDMaster12** [comment]
@Mandalorian007 
Disabled scale when called from combatantList

If you think this is all good, then I am ready to merge.

**Pull request merged**